examples/data-sources/maas_boot_source_selection/data-source.tf

```
@@ -0,0 +1,8 @@
+data "maas_boot_source" "default" {}
+
+data "maas_boot_source_selection" "default" {
+  boot_source = maas_boot_source.default.id
```

This one should be fixed, and the docs should be regenerated.

```suggestion
  boot_source = data.maas_boot_source.default.id
```